package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "userscripts",
```

Comment:
Let call this 

```suggestion
  "name": "jerone's userscripts",
```

Reply:
Done in 223fe958a669589d092187bd00f2f0d28b95007e